docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/MaxFilesExample.tsx

```diff
@@ -4,6 +4,7 @@ export const MaxFilesExample = () => {
   return (
     <FileUploader
       maxFiles={3}
+      variation="drop"
```

Comment:
Changing most of these examples to use the drop variation. This allows users and us to test the behavior of dropping files as well as opening the file browser.